spec/order_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'order'
+require 'menu'
+
+describe Order do
+  subject(:order){described_class.new}
+  let(:message){double(:message)}
+
+  describe '#show_menu' do
+   
+   it 'should list availble dishes' do
+    expect(order.show_menu).to eq(order.tidy_menu)
+   end
+  end
+
+  describe '#choose' do
+    it 'chooses a pizza' do
+      expect(order).to respond_to(:choose).with(2).argument
+    end
+
+    it 'matches with an item' do
+      allow(order).to receive(:choose) {"Margherita"}
+      expect(MENU).to include(subject.choose)
+    end
+  end
+
+  describe 'ordered_items' do
+
+    it 'returns array of orderd items' do
+        expect(order.ordered_items).to be_a Array
+      end
```

Comment:
Lint/BlockAlignment: `end` at 30, 6 is not aligned with `it 'returns array of orderd items' do` at 28, 4.